include/physics/analysis.h

```diff
@@ -212,7 +212,7 @@ namespace analysis{
     static int count = 0;
     const int screen_length = 40;
     if (out_screen_every > 0 || physics::iteration % out_screen_every == 0) {
-      (++count-1)%screen_length  ||
+      (++count-1)%screen_length;
```

Comment:
I'd like to keep the '||'. This ensures that the "# iteration ... time" thingy is printed only once per screen_length as opposed to output every iteration.

Reply:
It does not compile with the || anymore. 
We need to change to if/else. 

I just figure another problem, I will remove this pull request.